test/eth.accounts.signTransaction.js

```diff
@@ -757,6 +759,7 @@ describe("eth", function () {
                     })
                     .catch(e => {
                         console.log(i, e)
+                        done(e);
```

Comment:
What's the need for this change as well?

Reply:
If test is failing due to any assertion, it must fail not timeout. Skipping `done` in case of error was resulting the test to timeout.